recipes/computer_vision/object_detection/README.md

```diff
@@ -1,58 +1,91 @@
+This is a well performant model with an Apache-2.0 license.
+It's simple to download a pre-converted whisper model from [huggingface.co](https://huggingface.co)
+here: https://huggingface.co/facebook/detr-resnet-101.
+
+You can use the `download-model-facebook-detr-resnet-101` make target in the `model_servers/object_detection_python` directory download and move the model for you:
+
+```bash
+# from path model_servers/object_detection_python from repo containers/ai-lab-recipes
+ make download-model-facebook-detr-resnet-101
+```
 
-If you'd like to download models locally so that they are not pulled each time the container restarts, you can use the following python snippet to a model to your `models/` directory.  
+Or you can manually use the [python script](../../../convert_models/download_huggingface.py) we provide in the [`convert_models` directory](../../../convert_models/) to download the model, and then move it to the models directory.
 
-```python
-from huggingface_hub import snapshot_download
+## Build the Model Service
 
-snapshot_download(repo_id="facebook/detr-resnet-101",
-                  revision="no_timm",
-                local_dir="<PATH_TO>/locallm/models/vision/object_detection/facebook/detr-resnet-101",
-                local_dir_use_symlinks=False)
+The You can build the Model Service from the [object_detection_python model-service directory](../../../model_servers/object_detection_python)
 
+```bash
+# from path model_servers/object_detection_python from repo containers/ai-lab-recipes
+make build
 ```
 
-When using a model other than the default, you will need to set the `MODEL_PATH` environment variable. Here is an example of running the model service with a local model:
+Checkout the [Makefile](../../../model_servers/object_detection_python/Makefile) to get more details on different options for how to build.
+
+## Deploy the Model Service
+
+The local Model Service relies on a volume mount to the localhost to access the model files. It also employs environment variables to dictate the model used and where its served. You can start your local Model Service using the following `make` command from `model_servers/object_detection_python` set with reasonable defaults:
 
 ```bash
- podman run -it --rm -p 8000:8000 -v <PATH/TO>/locallm/models/vision/:/locallm/models -e MODEL_PATH=models/object_detection/facebook/detr-resnet-50/ object_detection_service
+# from path model_servers/object_detection_python from repo containers/ai-lab-recipes
+make run
 ```
 
-## Build and run the client application
+By default the model service will use [`facebook/detr-resnet-101`](https://huggingface.co/facebook/detr-resnet-101), which has an apache-2.0 license. The model is relatively small, but it will be downloaded fresh each time the model server is started unless a local model is provided (see additional instructions below).  
```

Comment:
```suggestion
By default the model service will use [`facebook/detr-resnet-101`](https://huggingface.co/facebook/detr-resnet-101), which has an apache-2.0 license. The model is relatively small, but it will be downloaded fresh each time the model server is started unless a local model is provided.
```